site/data/dynamo.ts

```diff
@@ -184,24 +208,31 @@ export const insertPublishedDisruptionIntoDynamoAndUpdateDraft = async (
                 },
                 UpdateExpression: "SET publishStatus = :1",
                 ExpressionAttributeValues: {
-                    ":1": "PUBLISHED",
+                    ":1": status,
                 },
             },
         })) ?? [];
 
+    const putSiriTable =
+        status !== PublishStatus.pendingApproval
```

Comment:
I would make this `status === PublishStatus.published` in case we extend this to handle more statuses in the future since we only want to push to the siri in the case where the disruption is published

Reply:
done